sknn/backend/lasagne/mlp.py

```diff
@@ -276,6 +276,8 @@ def _array_to_mlp(self, array, nn):
         for layer, data in zip(nn, array):
             if data is None: continue
             weights, biases = data
+            weights = weights.astype(theano.config.floatX)
```

Comment:
Yeah, looks much better. I'd move this cast into `layer.W.set_value(weights)` because it's not too important and doesn't need to take up a whole line.